docs/source/tutorials/mantid_basic_course/connecting_data_to_instruments/index.rst

```diff
@@ -4,6 +4,21 @@
 Connecting Data To Instruments
 ==============================
 
+Mantid supports Neutron and Muon Science at many :ref:`collaborating facilities
+<https://github.com/mantidproject/governance/blob/main/facilities.md>`.
+
+Feel free to checkout the instrument sections of each facility website.
+
+|ANSTO_logo.gif| |CSNS_Logo_Short.png| |ess_logo_transparent_small.png| |ILL_logo.png|
+
+|
+|
+
+|ISIS_Logo_Transparent.gif| |Julich.jpg| |ORNL_logo.jpg| |Psi_logo.png|
```

Comment:
It might just be my screen, but the PSI logo seems to drop to the next line and appears to be part of the ISIS logo?

Reply:
Good point! That seems to happen when the browser is too narrow ( I just get 2 rows on mine). I'll add some whitespace to the top of the PSI image to make it look a bit better!